ColorStripe/Extensions/Array+Convenience.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  Array+Convenience.swift
+//  ColorStripe
+//
+//  Created by Dorian Grolaux on 03/06/2021.
+//
+
+import Foundation
+
+extension RandomAccessCollection {
```

Comment:
[https://github.com/apple/swift-collections](swift-collections) has OrderedSet type, maybe it time to use third-party dependencies? What do you think? 🤔

Reply:
with regards to my latest comment (debrief), because of the generic structure of the code, this will trickle down generics even further (ordered set requires the peripheral to be Hashable). Either that or I make the view model itself only accept hashable peripherals... It's all a big spaghetti of dependencies & generics 😓 and I'm not sure it should be this way... :/ If you have any suggestions or insight, would really appreciate it :D